tests/techmap/iopadmap.ys

```diff
@@ -97,3 +111,12 @@ select -assert-count 1 @oeb %co %co @iob %i
 select -assert-count 1 @iob %co %co @o2b %i
 select -assert-count 1 @iob %co %co t:$_NOT_ %i
 select -assert-count 1 @o3b %ci %ci t:$_NOT_ %i
+
+select -assert-count 2 f/t:obuft
```

Comment:
If you have some good idea on how to assert that 0 is in fact connected to the oe pin, please tell.

Reply:
Perhaps look at `sat`? You can either get it to prove that a signal is a specific value, or you can throw some `assert`s in?

Reply:
The signal is a constant wired straight to a cell port though, how would I get at it?

Reply:
Ah, you're right, sorry! I don't think there's an elegant way then.

Reply:
Was traipsing through the Yosys docs when I stumbled across this: http://www.clifford.at/yosys/cmd_hilomap.html -- sounds like it should help?

Reply:
Ohhhh, that would work, thanks for the idea!